pkg/resources/row_access_policy.go

```diff
@@ -223,37 +155,29 @@ func ReadRowAccessPolicy(d *schema.ResourceData, meta interface{}) error {
 // UpdateRowAccessPolicy implements schema.UpdateFunc.
 func UpdateRowAccessPolicy(d *schema.ResourceData, meta interface{}) error {
 	db := meta.(*sql.DB)
-
-	rowAccessPolicyID, err := rowAccessPolicyIDFromString(d.Id())
-	if err != nil {
-		return err
-	}
-
-	dbName := rowAccessPolicyID.DatabaseName
-	schema := rowAccessPolicyID.SchemaName
-	policyName := rowAccessPolicyID.RowAccessPolicyName
-
-	builder := snowflake.RowAccessPolicy(policyName, dbName, schema)
+	ctx := context.Background()
+	client := sdk.NewClientFromDB(db)
+	id := helpers.DecodeSnowflakeID(d.Id()).(sdk.SchemaObjectIdentifier)
 
 	if d.HasChange("comment") {
 		comment := d.Get("comment")
 		if c := comment.(string); c == "" {
```

Comment:
is an unset comment the same as setting it to ""?

Reply:
That's a good question. I think this is one of the topics that we need to discuss globally with SNOW-867235. For me: if there is UNSET supported, we should use it. It may differ between different parameters though.